public/app/features/explore/RichHistory/RichHistoryQueriesTab.test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Change of SortOrder import as we have moved LogsSortOrder to grafana/data and moved SortOrder for Rich history to app/core/utils/richHistory.